#
# Submitted-By: Erwin Rol Mar 19 2006
# Committed-By: Erwin Rol
#
# Error:
# Select ALSA as default
#
# Description:
# Patch taken from Fedora Core 5 RPM
#
# State:
# Unknown
#

Index: SDL-1.2.9/src/audio/SDL_audio.c
===================================================================
--- SDL-1.2.9/src/audio/SDL_audio.c	(revision 58)
+++ SDL-1.2.9/src/audio/SDL_audio.c	(working copy)
@@ -40,6 +40,15 @@
 
 /* Available audio drivers */
 static AudioBootStrap *bootstrap[] = {
+#ifdef ALSA_SUPPORT
+	&ALSA_bootstrap,
+#endif
+#ifdef ARTSC_SUPPORT
+	&ARTSC_bootstrap,
+#endif
+#ifdef ESD_SUPPORT
+	&ESD_bootstrap,
+#endif
 #ifdef OPENBSD_AUDIO_SUPPORT
 	&OPENBSD_AUDIO_bootstrap,
 #endif
@@ -47,9 +56,6 @@
 	&DSP_bootstrap,
 	&DMA_bootstrap,
 #endif
-#ifdef ALSA_SUPPORT
-	&ALSA_bootstrap,
-#endif
 #ifdef QNXNTOAUDIO_SUPPORT
 	&QNXNTOAUDIO_bootstrap,
 #endif
@@ -59,12 +65,6 @@
 #ifdef DMEDIA_SUPPORT
 	&DMEDIA_bootstrap,
 #endif
-#ifdef ARTSC_SUPPORT
-	&ARTSC_bootstrap,
-#endif
-#ifdef ESD_SUPPORT
-	&ESD_bootstrap,
-#endif
 #ifdef NAS_SUPPORT
 	&NAS_bootstrap,
 #endif
